.gitignore

```diff
@@ -0,0 +1,28 @@
+#################
+## Eclipse
+#################
+
+*.pydevproject
+.project
+.metadata
+bin/
+tmp/
+*.tmp
+*.bak
+*.swp
+*~.nib
+local.properties
+.classpath
+.settings/
+.loadpath
+/target/
+.settings/org.eclipse.core.resources.prefs 
+test-output/old/Default suite/methods-alphabetical.html
+/test-output/ 
+
```

Comment:
Line 19 is a duplicate, given line 16. Line 20 is a duplicate given line 21. Line 21 should be "test-output/. Line 18 should be "target/

I think all of the files in "target/" are auto-generated, so you should `git rm -r --cached target/*` as well